lib/matplotlib/figure.py

```diff
@@ -2372,10 +2372,22 @@ def __init__(self,
         # pickling.
         self._canvas_callbacks = cbook.CallbackRegistry(
             signals=FigureCanvasBase.events)
-        self._button_pick_id = self._canvas_callbacks.connect(
+        connect = self._canvas_callbacks.connect
```

Comment:
Should we be using `_connect_pickable` here instead?

Reply:
Indeed, fixed.